logger_windows.go

```diff
@@ -1,3 +1,5 @@
+// +build windows
```

Comment:
This is redundant, the go tool uses the +build tags or the _windows

Reply:
Ack.  Thought this will make it more explicit.